assets_management/i18n/it.po

```diff
@@ -614,7 +614,7 @@ msgstr "Dismissione bene: "
 #: code:addons/assets_management/models/asset_depreciation_line.py:389
 #, python-format
 msgid "Asset: "
-msgstr "Bene ammortizzabile"
+msgstr "Bene ammortizzabile: "
```

Comment:
Una nota al volo. Se la traduzione vuole essere `Bene ammortizzabile` le corrispondenti stringhe in inglese adrebbero modificate in `Depreciable Asset`

Come glossario di riferimento può essere preso questo:
http://multilex.it/wp-content/uploads/2016/01/Dizionario-Tributario-ITALIANO-INGLESE-Prima-Edizione-MultiLex.pdf